core/services/relay/evm/chain_components_test.go

```diff
@@ -228,17 +230,51 @@ func TestChainReader_HealthReport(t *testing.T) {
 }
 
 func TestChainComponents(t *testing.T) {
-	tests.SkipFlakey(t, "https://smartcontract-it.atlassian.net/browse/BCFR-1083")
 	t.Parallel()
-	it := &EVMChainComponentsInterfaceTester[*testing.T]{Helper: &helper{}}
-	// TODO, generated binding tests are broken
-	it.DisableTests([]string{interfacetests.ContractReaderGetLatestValue})
-	it.Init(t)
-
+	// shared helper so all tests can run efficiently in parallel
+	helper := &helper{}
+	helper.Init(t)
+	deployLock := sync.Mutex{}
 	// add new subtests here so that it can be run on real chains too
-	RunChainComponentsEvmTests(t, it)
-	RunChainComponentsInLoopEvmTests[*testing.T](t, commontestutils.WrapContractReaderTesterForLoop(it))
-	RunChainComponentsInLoopEvmTests(t, WrapContractReaderTesterWithBindings(t, it))
+	t.Run("RunChainComponentsEvmTests", func(t *testing.T) {
+		t.Parallel()
+		it := &EVMChainComponentsInterfaceTester[*testing.T]{Helper: helper, DeployLock: &deployLock}
+		// These tests are broken in develop as well, so disable them for now
```

Comment:
CC @poopoothegorilla  Can you create an epic for general Chain Components work that is not urgent, but has to be done? Maybe just have 1 for now and then have them per chain